backend/fms_core/migrations/0022_v3_4_0.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.1 on 2021-08-04 17:59
+
+from django.conf import settings
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import re
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('fms_core', '0021_v3_3_0'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Project',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True, help_text='Date the instance was created.')),
+                ('updated_at', models.DateTimeField(auto_now=True, help_text='Date the instance was modified.')),
+                ('deleted', models.BooleanField(default=False, help_text='Whether this instance has been deleted.')),
+                ('name', models.CharField(help_text='The name of the project.', max_length=200, unique=True, validators=[django.core.validators.RegexValidator(re.compile('^[a-zA-Z0-9.\\-_]{1,200}$'))])),
+                ('principal_investigator', models.CharField(blank=True, help_text='The principal investigator of the project.', max_length=200)),
+                ('requestor_name', models.CharField(blank=True, help_text='The name of the requestor of the project.', max_length=200)),
+                ('requestor_email', models.CharField(blank=True, help_text='The email of the requestor of the project.', max_length=200, validators=[django.core.validators.EmailValidator()])),
+                ('targeted_end_date', models.DateTimeField(blank=True, help_text='Targeted date to conclude the project.')),
+                ('status', models.CharField(choices=[('Ongoing', 'Ongoing'), ('Completed', 'Completed')], help_text='The status of the project.', max_length=20)),
```

Comment:
If we do not get a clear list of status from the users, i would add personally an "On-Hold" status.

Reply:
So we would have Ongoing, Completed and On-Hold?

Reply:
Maybe this is something we should discuss once we have the users feedback ... i just noted this here to share and see what you and Jasmine think about this.